assets/js/langRedirector.js

```diff
@@ -0,0 +1,36 @@
+// Wrap all this to not pollute global namespace
+{
+    // Generate an array of languages from page
+    const JDM_LANGS = [...document.querySelectorAll('#language-dropdown > ul > li')].map(el => el.classList[0]);
+
+
+    function redirectTo(lang) {
+        const link = document.querySelector(`#language-dropdown > ul > li.${lang} > a`);
+        if (link)
+            link.click();
```

Comment:
Since our language URL is very simple, I'd rather set `window.location` to perform this redirection.
```suggestion
		if (lang) {
			window.location = `${(new URL(window.location)).origin}/${lang}`;
		}
```

Reply:
Not as simple. At least, we have `en` that points to `/`:
```suggestion
        if (lang) {
            let path = lang;
            if (lang === 'en')
                path = '';
            location = location.origin+'/'+path;
        }
```